block/provider/attach_volume_test.go

```diff
@@ -193,12 +193,11 @@ func TestAttachVolumeForInvalidSession(t *testing.T) {
 	assert.NotNil(t, uc)
 	assert.NotNil(t, sc)
 	assert.Nil(t, err)
-	expectedError := "{Code:InvalidServiceSession, Type:RetrivalFailed, Description:The Service Session was not found due to error while generating IAM token., BackendError:IAM token exchange request failed, RC:500}"
+	//expectedError := "{Code:InvalidServiceSession, Type:RetrivalFailed, Description:The Service Session was not found due to error while generating IAM token., BackendError:IAM token exchange request failed., BackendError:IAM token exchange request failed, RC:500}"
```

Comment:
why are we not checking the expected error and commenting ?

Reply:
Updated to compare the actual error. Initially, the expected Error was different, that is because of some changes in UT. Now, it is fixed here and in other places also.